.eslintrc-base.js

```diff
@@ -23,34 +23,13 @@ module.exports = {
   },
 
   settings: {
-    react: {
-      version: 'detect',
-    },
     'import/resolver': {
       alias: [
-        ['~components', './src/components'],
-        ['~styles', './src/styles'],
-        ['~hooks', './src/hooks'],
-        ['~config', './src/config'],
-        ['~lib', './src/lib'],
+        ['~', './'],
```

Comment:
I'd move `import/resolver` configuration into node and web specific files, since FE and BE use different combinations. Apart from that, this config doesn't work for me, I have to have `map` object like;

```
"alias": {
        "map": [
          ["@module", "./src/modules"],
          ["@common", "./src/common"],
          ["@util", "./src/util"]
        ],
        "extensions": [".js", ".ts"]
      }
```